docusaurus/docs/troubleshooting.md

```diff
@@ -62,7 +62,7 @@ If you're using mounts to access your files, it's likely that webpack isn't dete
 #### What you can do:
 
 - If you're editing your code from a native Windows app (such as VS Code), you need to manually rerun `yarn build` every time you want to see a change in your plugin.
-- Use [webpack `watchOption` with `poll`](https://webpack.js.org/configuration/watch/#watchoptionspoll) in your project. Find instructions on how to extend webpack configuration in our [documentation](https://grafana.com/developers/plugin-tools/create-a-plugin/extend-configurations).
+- Use [webpack `watchOption` with `poll`](https://webpack.js.org/configuration/watch/#watchoptionspoll) in your project. Find instructions on how to extend webpack configuration in our [documentation](./get-started/set-up-development-environment.mdx#extend-configurations).
```

Comment:
This has been built into the [create-plugin webpack](https://github.com/grafana/plugin-tools/blob/main/packages/create-plugin/templates/common/.config/webpack/webpack.config.ts#L241-L246) configs for ~10 months now. I would suggest asking devs to run the `create-plugin update` command over suggesting to customise the config.

Reply:
```suggestion
- Ensure you are running the latest recommended webpack configuration by running the [update](./migration-guides/update-create-plugin-versions.mdx) command in `create-plugin`.
```
How's this

Reply:
works for me.